README.md

```diff
@@ -1,2 +1,13 @@
 # go-wazimap-js
-A USSD interface to the Wazimap API
+A USSD interface to the Wazimap API (www.wazimap.co.za) 
+
+USSD Number (* 120 * 8864 * 1601 #)
```

Comment:
I think you can escape `*` using `\*` so the number would become `\*120\*8864\*1601\#`.